/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py:208: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use-env is set by default in torchrun.
If your script expects `--local-rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  main()
W0807 15:51:01.089672 126691882321728 torch/distributed/run.py:779] 
W0807 15:51:01.089672 126691882321728 torch/distributed/run.py:779] *****************************************
W0807 15:51:01.089672 126691882321728 torch/distributed/run.py:779] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
W0807 15:51:01.089672 126691882321728 torch/distributed/run.py:779] *****************************************
/home/hslee/FINE-FPN/mmengine/mmengine/optim/optimizer/zero_optimizer.py:11: DeprecationWarning: `TorchScript` support for functional optimizers is deprecated and will be removed in a future PyTorch release. Consider using the `torch.compile` optimizer instead.
  from torch.distributed.optim import \
/home/hslee/FINE-FPN/mmengine/mmengine/optim/optimizer/zero_optimizer.py:11: DeprecationWarning: `TorchScript` support for functional optimizers is deprecated and will be removed in a future PyTorch release. Consider using the `torch.compile` optimizer instead.
  from torch.distributed.optim import \
Traceback (most recent call last):
  File "mmdetection/tools/train.py", line 123, in <module>
    main()
  File "mmdetection/tools/train.py", line 68, in main
    cfg = Config.fromfile(args.config)
  File "/home/hslee/FINE-FPN/mmengine/mmengine/config/config.py", line 458, in fromfile
    lazy_import is None and not Config._is_lazy_import(filename):
  File "/home/hslee/FINE-FPN/mmengine/mmengine/config/config.py", line 1662, in _is_lazy_import
    with open(filename, encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py'
Traceback (most recent call last):
  File "mmdetection/tools/train.py", line 123, in <module>
    main()
  File "mmdetection/tools/train.py", line 68, in main
    cfg = Config.fromfile(args.config)
  File "/home/hslee/FINE-FPN/mmengine/mmengine/config/config.py", line 458, in fromfile
    lazy_import is None and not Config._is_lazy_import(filename):
  File "/home/hslee/FINE-FPN/mmengine/mmengine/config/config.py", line 1662, in _is_lazy_import
    with open(filename, encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py'
W0807 15:51:05.417922 126691882321728 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 112401 closing signal SIGTERM
E0807 15:51:05.732485 126691882321728 torch/distributed/elastic/multiprocessing/api.py:833] failed (exitcode: 1) local_rank: 0 (pid: 112400) of binary: /home/hslee/anaconda3/envs/openmmlab/bin/python
Traceback (most recent call last):
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py", line 208, in <module>
    main()
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/typing_extensions.py", line 2853, in wrapper
    return arg(*args, **kwargs)
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py", line 204, in main
    launch(args)
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py", line 189, in launch
    run(args)
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/run.py", line 892, in run
    elastic_launch(
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 133, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/hslee/anaconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
mmdetection/tools/train.py FAILED
------------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-08-07_15:51:05
  host      : pascal
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 112400)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
